Damage.cs

```diff
@@ -17,49 +16,54 @@ public EDamageTypes DamageType
                 return damageType;
             }
         }
-        #endregion
 
-        #region TargetAttribute
         public struct Target
         {
             public float Armor;
 
             public ERA ERAData;
```

Comment:
Rename `ERAData` to `EraData`. Acronyms greater than 2 characters need to be camel case like any other variable. e.g. HttpExample (*instead* of HTTPExample).